src/routes/+page.server.ts

```diff
@@ -26,11 +27,19 @@ export const actions: Actions = {
       steps,
       technical
     }) => {
-      const priorityLabel: Record<string, string> = {
-        low: '🟢  **Low**',
-        medium: '🟡  **Medium**',
-        high: '🟠  **High**',
-        critical: '🔴  **Critical**'
+      const getPriorityLabel = (priority: string) => {
```

Comment:
instead of typing the `priority` as a string which can be any string. You should be more strict to allow only the `priorityType` values

```suggestion
      const getPriorityLabel = (priority: `${priorityType}`) => {
```
I don't like to use enums, but looks like this is the way to do it 🤷‍♂️